kitsune/sumo/utils.py

```diff
@@ -254,32 +254,34 @@ def _wr(self, s):
         sys.stdout.flush()
 
 
-def is_ratelimited(request, name, rate, method=["POST"], skip_if=lambda r: False):
+def is_ratelimited(request, name, rate, method="POST"):
     """
-    Reimplement ``ratelimit.helpers.is_ratelimited``, with sumo-specific details:
+    Wraps ``ratelimit.core.is_ratelimited`` with sumo-specific details:
 
     * Always check for the bypass rate limit permission.
     * Log times when users are rate limited.
     * Always uses ``user_or_ip`` for the rate limit key.
+    * Adds the Boolean attribute "limited" to the request.
     """
-    if skip_if(request) or request.user.has_perm("sumo.bypass_ratelimit"):
+    if request.user.has_perm("sumo.bypass_ratelimit"):
```

Comment:
You could refactor the next block to:

```python
if request.user.has_perm("sumo.bypass_ratelimit"):
    return False
if (limited := is_ratelimited_core(...):
``` 
This will remove the `else` and an `if` statement, simplifying the code a bit

Reply:
Nice! I haven't used the "walrus" operator much yet.